css/site.styl

```diff
@@ -103,6 +108,58 @@ img
   height: auto
 
 
+// FIXED ELEMENTS
+
+#header
+  position: fixed
+  top: 0
+  left: 0
+  right: 0
+
+#footer
+  display: none
+  position: fixed
+  bottom: 0
+  left: 0
+  right: 0
+
+.page, .single
+  #footer
+    display: block
+
+#gradient
+  position: fixed
+  top: 0
+  right: 0
+  bottom: 0
+  width: 10%
+  background-image: -moz-linear-gradient(left,  rgba(255,255,255,0) 0%, rgba(216,216,216,0.33) 33%, rgba(63,63,63,0.7) 70%, rgba(19,19,19,1) 100%)
+  background-image: -webkit-linear-gradient(left,  rgba(255,255,255,0) 0%,rgba(216,216,216,0.33) 33%,rgba(63,63,63,0.7) 70%,rgba(19,19,19,1) 100%)
+  background-image: linear-gradient(to right,  rgba(255,255,255,0) 0%,rgba(216,216,216,0.33) 33%,rgba(63,63,63,0.7) 70%,rgba(19,19,19,1) 100%)
+
+#loading
+  position: fixed
+  top: 0
+  right: 0
+  bottom: 0
+  left: 0
+  z-index: -999
```

Comment:
doesnt do anything yet. my conception is that we will probably hide #main-content and on doc ready fade that in and fade out #loading